### PR TITLE
Add telemetry hook

### PR DESCRIPTION
**Pull request by rihorn2**
Add telemetry hook and interface. Call if the inputs are of inconsistent dimensions.

### CODE REVIEW

File: visualization/dashboard/src/MLIDashboard/Interfaces/IExplanationDashboardProps.ts [hunk 1]
```diff
@@ -37,6 +38,25 @@ export interface IExplanationDashboardProps {
     ) => Promise<any[]>;
 }
 
+export enum TelemetryLevels {
```
Comment (wamartin-aml) [inline visualization/dashboard/src/MLIDashboard/Interfaces/IExplanationDashboardProps.ts:41]: My first reaction is that these should maybe go in their own file, but on more reflection it kinda depends on how you've laid out other stuff. Up to you.

File: visualization/dashboard/src/MLIDashboard/Interfaces/IExplanationDashboardProps.ts [hunk 2]
```diff
@@ -37,6 +38,25 @@ export interface IExplanationDashboardProps {
...
+    error = 'Error',
+    warning = 'Warning',
+    info = 'Info',
+    trace = 'Trace'
+}
+
+export type TelemetryLevel =
```
Comment (marksoper) [inline visualization/dashboard/src/MLIDashboard/Interfaces/IExplanationDashboardProps.ts:48]: In the first iteration, it looks like Error is the only Telemetry type implemented right?

What do you have in mind for the other types?

Comment (rihorn2) [reply visualization/dashboard/src/MLIDashboard/Interfaces/IExplanationDashboardProps.ts:48]: To be clear, the dashboard is not implementing anything for telemetry, thats what the hook is for, the hook passed in is the implementation. So Studio would pass in a function that did whatever made sense for Azure telemetry on an error (user errors in this case) versus a warning versus tracking user actions.

File: visualization/dashboard/src/MLIDashboard/Interfaces/IExplanationDashboardProps.ts (hunk 1 above)
Comment (rihorn2) [reply visualization/dashboard/src/MLIDashboard/Interfaces/IExplanationDashboardProps.ts:41]: Yes, adding new files for each of the different interfaces may be clearer for future developers.

**Pull request merged**